src/components/Typography/TypographyBase/index.tsx

```diff
@@ -1,9 +1,13 @@
+import { ElementType } from "react";
+
 import {
     Typography as MaterialTypography,
     TypographyProps as MaterialTypographyProps,
 } from "@material-ui/core";
 
-export interface TypographyBaseProps extends MaterialTypographyProps {}
+export interface TypographyBaseProps extends MaterialTypographyProps {
+    component?: ElementType<any>;

```

Comment:
I don't think this should be any

Reply:
Can you give me some help? I cannot find the suitable type for ElementType generic

Reply:
You can google meet me whenever I'm free

Reply:
okay I will ask directly in the messenger.